.env

```diff
@@ -3,9 +3,9 @@
 REACT_APP_API_URL=http://localhost:4000
 REACT_APP_JORE_API_URL=https://dev.kartat.hsl.fi/jore/graphql
 REACT_APP_CLIENT_ID=7833861618225795
-REACT_APP_CLIENT_SECRET=
 REACT_APP_REDIRECT_URI=http://localhost:3000
-HSL_TESTING_HSLID_USERNAME=
-HSL_TESTING_HSLID_PASSWORD=
-DOMAINS_ALLOWED_TO_GENERATE=
-REACT_APP_NAMESPACE=hsl-kartta
```

Comment:
This value (REACT_APP_NAMESPACE) is not set in stage env-file

Reply:
Done